src/collective/excelexport/view.py

```diff
@@ -140,17 +146,15 @@ def _format_render(self, render):
         elif not isinstance(render, unicode):
             render = unicode(render)
 
-        return render.encode(encoding=self.encoding)
+        try:
+            return render.encode(encoding=self.encoding)
+        except UnicodeEncodeError:
+            # try default encoding
```

Comment:
ok, maybe you should be explicit and try render.encode(encoding="utf-8") as fallback instead of str(render) which normally is an ascii encoding unless you changed sys.setdefaultencoding?